packages/serverless/src/integrations/aws_lambda.ts

```diff
@@ -0,0 +1,248 @@
+        Name: nodeJs,
+        // global.process.version return the version of node
+        Version: global.process.version,
+      });
+    }
+
+    /**
+     * This function sets Additional Lambda Parameters which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    function setAdditionalLambdaParameters(scope: Scope): void {
+      const remainingTimeInMillisecond: number = setupOnceContext.awsContext.getRemainingTimeInMillis() as number;
+      const executionTime: number = setupOnceContext.configuredTimeInMilliseconds - remainingTimeInMillisecond;
+
+      scope.setExtra('lambda', {
+        aws_request_id: setupOnceContext.awsContext.awsRequestId,
+        function_name: setupOnceContext.awsContext.functionName,
+        function_version: setupOnceContext.awsContext.functionVersion,
+        invoked_function_arn: setupOnceContext.awsContext.invokedFunctionArn,
+        execution_duration_in_millis: executionTime,
+        remaining_time_in_millis: setupOnceContext.awsContext.getRemainingTimeInMillis(),
+      });
+    }
+
+    /**
+     * This function sets Cloud Watch Logs data which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    function setCloudwatchLogsData(scope: Scope): void {
+      scope.setExtra('cloudwatch logs', {
+        log_group: setupOnceContext.awsContext.logGroupName,
+        log_stream: setupOnceContext.awsContext.logStreamName,
+        /**
+         * process.env.AWS_REGION -  this parameters given the AWS region
+         * process.env.AWS_LAMBDA_FUNCTION_NAME - this parameter provides the AWS Lambda Function name
+         */
+        url: `https://${process.env.AWS_REGION}.console.aws.amazon.com/cloudwatch/home?region=${process.env.AWS_REGION}#logsV2:log-groups/log-group/$252Faws$252Flambda$252F${process.env.AWS_LAMBDA_FUNCTION_NAME}`,
+      });
+    }
+
+    /**
+     * This function sets tags which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    function setTags(scope: Scope): void {
+      scope.setTag('runtime', `${nodeJs} ${global.process.version}`);
+      scope.setTag('transaction', setupOnceContext.awsContext.functionName);
+      scope.setTag('runtime.name', nodeJs);
+      scope.setTag('server_name', String(process.env._AWS_XRAY_DAEMON_ADDRESS));
+      scope.setTag('url', 'awslambda:///' + setupOnceContext.awsContext.functionName);
+    }
+
+    /**
+     * This function return event within hub
+     * @param eventId  - contains event Id of the event
+     * @hidden
+     */
+    function invokeEvent(eventId: String): void {
+      if (hub && hub.getIntegration(AWSLambda)) {
+        hub.withScope((scope: Scope) => {
+          scope.addEventProcessor(event => {
+            event.event_id = eventId.toString();
+            return event;
+          });
+        });
+      }
+    }
```

Comment:
Sorry, I didn't quite follow what we intend here. It seems like a helper to overwrite event ids, why do we need that?

Reply:
Relevant changes have been done and the code has been removed in latest code